### PR TITLE
Update sol1.c

### PR DESCRIPTION
**Pull request by ilkerkosaroglu**
fixed infinite loop ? (some implementations of the function were not halting)

### CODE REVIEW

File: project_euler/Problem 03/sol1.c [hunk 1]
```diff
@@ -16,7 +16,7 @@ int isprime(int no) {
 		return 0;
 	}
```
Comment (TigerZheng) [inline project_euler/Problem 03/sol1.c]: else{
}

File: project_euler/Problem 03/sol1.c [hunk 2]
```diff
@@ -16,7 +16,7 @@ int isprime(int no) {
...
 	sq = ((int)(sqrt(no))) + 1;
-	for (int i = 3; i < sq; i + 2) {
+	for (int i = 3; i < sq; i += 2) {
 		if (no%i == 0) {
 			return 0;
 		}
```
Comment (TigerZheng) [inline project_euler/Problem 03/sol1.c]: else{
}

File: project_euler/Problem 5/solution1.c [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		
+	}
+}
+int main(){
+	int smallest=1,x,number,last=0;
+	scanf("%d",&number);
+	
+	int primes[100];
+	int powerList[100];
+	for (int i=0;i<100;i++){
```
Comment (christianbender) [inline project_euler/Problem 5/solution1.c]: Here you can use [memset](https://www.tutorialspoint.com/c_standard_library/c_function_memset.htm)

**Comment by christianbender** [comment]
@ilkerkosaroglu Please seperate your pull request. And make the changes for problem3/sol1.c again

File: project_euler/Problem 03/sol1.c [hunk 4]
```diff
@@ -16,7 +16,7 @@ int isprime(int no) {
 		return 0;
 	}
 	sq = ((int)(sqrt(no))) + 1;
-	for (int i = 3; i < sq; i + 2) {
+	for (int i = 3; i < sq; i += 2) {
```
Comment (christianbender) [inline project_euler/Problem 03/sol1.c]: I don't see this changes in your master branch on my machine.

**Pull request closed without merging**